examples/text_summarization/summarization_evaluation.ipynb

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
<p>adjcent&nbsp;typo</p><p>sentence level&nbsp;-&gt; sentence-level&nbsp;</p><p>Would it better to use <a href="https://github.com/bheinzerling/pyrouge" target="_blank">https://github.com/bheinzerling/pyrouge</a> instead of <a href="https://github.com/bheinzerling/pyrouge/tree/master/pyrouge" target="_blank">https://github.com/bheinzerling/pyrouge/tree/master/pyrouge</a>?</p><p>other languages-&gt; other natural languages?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/526/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Fixed 1, 2, and 3. For 4, I think it's clear from the context that I don't mean programming languages.